src/main/java/seedu/address/model/AddressBook.java

```diff
@@ -83,6 +83,10 @@ public boolean hasPerson(Person person) {
         return persons.contains(person);
     }
 
+    /**
+     * Adds a person to the address book.
+     * The person must not already exist in the address book.
+     */
```

Comment:
This commit content doesn't match commit message of "Add UniqueSessionList in Address Book"